### ISSUE TITLE
Run the auto-correct flags only for specified files and lines

### ISSUE THREAD

**Issue by axcdnt** [rubocop/rubocop#10453]
## Is your feature request related to a problem? Please describe.

I'm working on a code base that until very recently, wasn't using RuboCop. We've decided to apply it to make sure the team is using the same standards. To make things a bit easier, we're combining with [Pronto](https://github.com/prontolabs/pronto) gem, that basically helps by running RuboCop only for the modified files/lines.This way, we avoid huge non-related changes for the PR.

## Describe the solution you'd like

I've always used RuboCop's `-a` and `-A` flag for my own projects, but it always applies for the whole file. I'd like to have a way to specify a bunch of files and their respective lines as the input for auto-correct:

```
bundle exec rubocop -a /
    file_a.rb:33 /
    file_b.rb:120 /
```

## Describe alternatives you've considered

N/A

## Additional context

I guess this would be very handy for teams gradually starting to use RuboCop. 

Thanks a ton for the great work with RuboCop! 🤗 🚀

**Comment by crokobit** [comment]
Need this feature too!

**Comment by ballerolu** [comment]
In the same boat for my team. We're gradually enforcing Rubocop. It would be nice to run auto-correct for minor rules on only changed lines (same way Pronto runs check on only changed lines).

**Comment by jonas054** [comment]
My suggestion is that you request this feature in [pronto](https://github.com/prontolabs/pronto). It's almost there already. By adding

```yaml
rubocop:
  suggestions: true
```

in `.pronto.yml` you get output like:

```
activerecord/lib/active_record/transactions.rb:295 W: Layout/LeadingCommentSpace: Missing space after `#`.

``suggestion
    def save(**) # :nodoc:
``
```

so all that's missing is for `pronto` to write those suggestions to the files.
